### DISCUSSION TITLE
How can we validate a container running on a remote host is isolated by kata-container?

### DISCUSSION THREAD

**Discussion by zibinpan** [kata-containers/kata-containers#9440] [General]
In such a scenario: A remote server (untrusted) claims that he provide a service in a trusted container, which is created by a public trusted image and isolated by kata-container and is running in a read-only mode (so that nobody can modify the container). However, as a user of this container, how can I validate this container? Specifically, when I connect to this container, how can I check if this container is really isolated by kata-container and really come from a trusted image?

**Comment by simonhammes** [comment]
I know this is a relatively old post, but I just stumbled upon this. You can check `dmesg | grep kata`.

Have you found an alternative approach?
